CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 
 # Changelog
 
+## Version 1.0.17
```

Comment:
Version 1.0.16 is not yet released. Feel free to add this to that section.

Reply:
Will do.